src/main/java/net/sf/jabref/BibDatabaseContext.java

```diff
@@ -65,7 +65,7 @@ public BibDatabaseContext(BibDatabase database, MetaData metaData, File file) {
         this(database, metaData, file, new Defaults());
     }
 
-    public BibDatabaseContext(Defaults defaults, DatabaseLocation location, String keywordSeparator) {
+    public BibDatabaseContext(Defaults defaults, DatabaseLocation location, Character keywordSeparator) {
```

Comment:
Ok, sounds reasonable. Let's hope that no one ever wants to use some weird multi-keyword separator.